app/Http/Requests/BatchRequest.php

```diff
@@ -26,9 +26,10 @@ public function rules()
         return [
             'start_day' => 'required',
             'stop_day' => 'required',
-            'workspace_id' => 'required',
-            'position_id' => 'required',
-            'program_id' => 'required',
+            'workspace_id' => 'required|numeric',
+            'position_id' => 'required|numeric',
+            'program_id' => 'required|numeric',
+            'subjects' => 'required'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/BatchRequest.php</br>**Line**: 32</br>Each line in an array declaration must end in a comma